Automattic/simplenote-ios
# Fastlane `match`: Migrate Code Signing from Google Cloud to S3

### Summary

This PR: 
- Migrates the code signing certificates and provisioning storage from Google Cloud to S3
- Uses the DRY'd code signing lane setup from WooCommerce iOS and WordPress iOS to better organize the code signing lanes
- Updates Fastlane from `2.216.0` to `2.220.0`
- Deletes the `Appfile` because the `team_id` is already read in the lanes from environment variables
- Deletes the `Matchfile` and moves that info into the appropriate lanes within the `Fastfile`
- Removes unneeded values from the example user `env` file

### Test
1. In your local `~/.simplenoteios-env.default` env file, add values for `MATCH_S3_ACCESS_KEY`, `MATCH_S3_SECRET_ACCESS_KEY`, and `MATCH_PASSWORD` from Mobile Secrets
2. Run `bundle exec fastlane update_certs_and_profiles`
3. Verify that it completes successfully
4. Run `bundle exec fastlane update_certs_and_profiles readonly:false`
5. Verify that it completes successfully (everything is up to date, so no changes will be made)

### Review
You can view the Installable build that uses the S3 `match` storage here in Buildkite: https://buildkite.com/automattic/simplenote-ios/builds/771#018ee8a0-ff7e-4505-87ee-53f09f0be1c7/481-503

### Release
These changes do not require release notes.
